### PR TITLE
Docs/nuxt rework

### PR DESCRIPTION
**Pull request by m0ksem**
Here we try to move our docs to nuxt and TEST HOW VUESITC WORKS WITH NUXT

Be ready for some interesting bugs. It's our test project. 

close https://github.com/epicmaxco/vuestic-ui/issues/934

Sub tasks:
- [x] i18n (require testing)
- [x] vuestic (required testing)
- [x] landing
- [ ] page configs - loading from folder on ssr.
   - [ ] need all blocks (we have a few easiest for now)
- [ ] layout 
  - [x] header
  - [x] sidebar
  - [ ] page content

Please, make sure you follow nuxt's project architecture.

### CODE REVIEW

File: packages/nuxt-docs/components/docs/DocsPage.vue [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import * as views from './blocks'
+
+const configName = 'test'
+
+const { getConfigAsync } = usePageConfig()
```
Comment (LighthouseKeeperYN) [inline packages/nuxt-docs/components/docs/DocsPage.vue:16]: usePageConfig is not imported

File: packages/nuxt-docs/modules/RouterView.vue [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <div>
+    <!-- Router view should be wrapper in div -->
+    <NuxtPage />
+    {{ }}
```
Comment (LighthouseKeeperYN) [inline packages/nuxt-docs/modules/RouterView.vue:5]: huh

File: packages/nuxt-docs/composables/useI18n.ts [hunk 3]
```diff
@@ -25,11 +25,20 @@ export const useI18n = () => {
...
     return translationString
-  } 
+  }
+
+  const route = useRoute()
+
+  const urlPrefix = computed(() => route.params.locale ? `/${route.params.locale}` : '')
+
+  const localizePath = (path: string) => {
+    return (`${urlPrefix.value}/${path}`).replace('//', '/')
```
Comment (LighthouseKeeperYN) [inline packages/nuxt-docs/composables/useI18n.ts:35]:
```suggestion
    return (`/${route.params.locale || ''}/${path}`).replace(/^\/+/, '/')
```

File: packages/nuxt-docs/composables/useI18n.ts [hunk 4]
```diff
@@ -25,11 +25,20 @@ export const useI18n = () => {
     }
 
     return translationString
-  } 
+  }
+
+  const route = useRoute()
+
+  const urlPrefix = computed(() => route.params.locale ? `/${route.params.locale}` : '')
```
Comment (LighthouseKeeperYN) [inline packages/nuxt-docs/composables/useI18n.ts:32]:
```suggestion
```

**Review by LighthouseKeeperYN — APPROVED** [review]

**Comment by m0ksem** [comment]
Looks like import.meta doesn't work in production builds, so we need some plugin that generates pages.

**Review by m0ksem — COMMENTED** [review]
Looks like some of PRs for 1.4.0 was pointed to this branch... So I will merge it in v1.4.0, but disable nuxt docs.

**Pull request merged**